### PR TITLE
Feat: Update action when simple payment motion finalized to include networkFee and amountLessFee

### PR DESCRIPTION
**Pull request by iamsamgibbs**
[See CDapp counterpart 2866](https://github.com/JoinColony/colonyCDapp/pull/2866)

### CODE REVIEW

File: src/handlers/motions/motionFinalized/helpers.ts [hunk 1]
```diff
@@ -353,3 +358,77 @@ export const updateColonyUnclaimedStakes = async (
...
+    const networkInverseFee = await getNetworkInverseFee();
+    if (!networkInverseFee) {
+      throw new Error(
+        'Network inverse fee not found. This is a bug and should be investigated.',
+      );
+    }
+
+    const amountWithFee = colonyAction.amount;
+
+    const amountLessFee = getAmountLessFee(amountWithFee, networkInverseFee);
```
Comment (mmioana) [inline src/handlers/motions/motionFinalized/helpers.ts:420]: I know it wasn't introduced in this PR, but maybe we could rename this helper function in `getAmountExcludingFee`?

**Review by mmioana — APPROVED** [review]
Was tested as part of the [CDapp PR](https://github.com/JoinColony/colonyCDapp/pull/2866)

**Review by bassgeta — CHANGES_REQUESTED** [review]
Looks all good, nice and straight forward!
I think we can just log the error instead of throwing it, but other than that, we're good to go :sunglasses:

**Review by bassgeta — APPROVED** [review]
Reapproving the error removal and rebase :+1: Let's gooooo :sunglasses:

**Review by rumzledz — APPROVED** [review]
Tested as part of #2866

**Pull request merged**